Chart/Chart.swift

```diff
@@ -291,6 +297,12 @@ class Chart: UIControl {
                 
                 if series.line {
                     drawLine(xValues: scaledXValues, yValues: scaledYValues, seriesIndex: index)
+                    if series.points {
+                        for i in 1..<scaledXValues.count {
```

Comment:
I won't use another loop here because it is resource expensive. I think the right method for adding the markers is the `drawLabelsAndGridOnXAxis`, i.e. where it is drawing the vertical lines for the x-axis.